src/renderer/components/swap/Swap.tsx

```diff
@@ -50,13 +56,14 @@ type SwapProps = {
   onConfirmSwap: (source: Asset, amount: AssetAmount, memo: string) => void
   poolDetails?: PoolDetails
   walletBalances?: O.Option<NonEmptyWalletBalances>
-  nativeTxFee: NativeFeeRD
   txWithState?: TxWithStateRD
   resetTx?: () => void
   goToTransaction?: (txHash: string) => void
   runeAsset: Asset
   activePricePool: PricePool
   PasswordConfirmation: React.FC<{ onSuccess: () => void; onClose: () => void }>
+  reloadFees?: () => void
+  fees?: SwapFeesRD
```

Comment:
Why not just `SwapFeesLD`?

Reply:
did not get you. 
'cuz this is `RD` not `LD` ? 🤔

Reply:
Yep, `SwapFeesRD` is fine - my bad - sorry!